Bash_scripts/HW23/README.md

```diff
@@ -0,0 +1,36 @@
+# Ansible. Do for ubuntu and centos hosts.
+
+###  1.	Difference between ansible module: command, shell and raw
+###  2.	Create a role hw23. Inside the role:
+    
+      a.	Create file /opt/hw23_test_file. Set 644 permissions
+    
+      b.	Do in 2 ways. Rename the file /opt/hw23_test_file to hw23_test_file_exists if file exists(/opt/hw23_test_file) otherwise print debug message that FILE_NAME does not exist.
```

Comment:
Did you do it in two ways?

Reply:
you can use Command as I did, and I suppose that you can still use the  module copy